Bugsnag/Payload/BugsnagStackframe.m

```diff
@@ -66,6 +70,90 @@ + (BugsnagStackframe *)frameFromDict:(NSDictionary *)dict
     }
 }
 
++ (NSArray<BugsnagStackframe *> *)stackframesWithCallStackSymbols:(NSArray<NSString *> *)callStackSymbols {
+    NSString *pattern = (@"^(\\d+)"             // Capture the leading frame number
+                         @" +"                  // Skip whitespace
+                         @"(\\S+)"              // Image name
```

Comment:
Hi, sorry if it's inappropriate to dig up an old pull request like this. I was trying to use this function to rebuild a stacktrace with some `callStackSymbols` I cached for an unhandled error and I believe I'm losing my apps frames because there is a space in our apps display name. For Example:

```
"0   ReactNative Test                     0x000000010fda7f1b RCTJSErrorFromCodeMessageAndNSError + 79"
```

Reply:
Thanks for reporting this - we are working on a fix: https://github.com/bugsnag/bugsnag-cocoa/pull/1036